infrastrucutre/vega-app-helm-chart/values.yaml

```diff
@@ -0,0 +1,39 @@
+# Default values for vega-app-helm-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+name: "vega-course-app"
+namespace: "vegait-training"
+
+postgres_config:
+  db: "stefandb"
+  user: "stefanuser"
+  password: "stefanpassword"
+  port: 5432
+  host: "vega-course-postgresql"
+
+service:
+  type: ClusterIP
+  port: 80
+  protocol: TCP
+  name: "http-service"
+
+ingress:
+  class: "alb"
+  scheme: internet-facing
+  type: ip
+  ssl_redirect: "443"
+  certificate_arn: "arn:aws:acm:eu-central-1:533267377331:certificate/9484c84e-481e-4907-aa6d-2e8851a3e946"
```

Comment:
values that change like url or cert arn should not be in git.
In this case **certificate_arn** is not even needed at all.